tests/js/spec/views/performance/table.spec.jsx

```diff
@@ -170,6 +171,24 @@ describe('Performance > Table', function () {
     await tick();
     wrapper.update();
     const firstRow = wrapper.find('GridBody').find('GridRow').at(0);
+    const transactionCell = firstRow.find('GridBodyCell').at(1);
+    expect(transactionCell.find('Link').prop('to')).toEqual({
+      pathname: '/organizations/org-slug/performance/summary/',
+      query: {
+        transaction: '/apple/cart',
+        project: '2',
+        environment: [],
+        statsPeriod: '14d',
+        start: '2019-10-01T00:00:00',
+        end: '2019-10-02T00:00:00',
+        query: 'event.type:transaction', // drops 'transaction:/api*' from the query
```

Comment:
We almost certainly want to drop `event.type:transaction` from the query as well as it is implied by the fact that we're in the performance product.

Reply:
I'll add a ticket to remove this and any other implicit filters to do as a follow up, so I can merge this without requiring a re-review! https://getsentry.atlassian.net/browse/VIS-1312